frontend/src/routes/(authed)/player/[id]/+page.svelte

```diff
@@ -84,10 +84,12 @@
     />
   {/if}
 
-  {#if data.stats.opponents.length > 0}
+  {#if data.opponents.length > 0}
     <Card.Root>
       <Card.Content class="flex flex-col p-0 md:p-6">
-        <h2 class="text-xl md:text-2xl">Most common opponents</h2>
+        <h2 class="px-4 py-3 text-xl md:p-0 md:text-2xl">
+          ⚔️ Most common opponents
+        </h2>
```

Comment:
We should properly soon make a Header component. We have this kind of h2 lots of places 😅

Reply:
Yep 😅